### PR TITLE
Introduce Stats Messaging (+ notifications on flow events)

### PR DESCRIPTION
**Pull request by sergii-iakovenko**
The changes:
- A new Stats Messaging module has been introduced.
- Rewrite StatsTopology to accept notification on flow updates instead of intercepting FL commands.
- Refactor StatsTopology to align with the actual system design (transport-service, etc.)

### CODE REVIEW

File: src-java/stats-topology/stats-messaging/src/main/java/org/openkilda/messaging/info/stats/RemoveFlowPathInfo.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+@JsonSerialize
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class RemoveFlowPathInfo extends InfoData {
+    private static final long serialVersionUID = 1L;
+
+    @JsonProperty("flow_id")
+    String flowId;
+
+    @JsonProperty("cookie")
+    long cookie;
```
Comment (surabujin) [inline src-java/stats-topology/stats-messaging/src/main/java/org/openkilda/messaging/info/stats/RemoveFlowPathInfo.java:45]: We should not loose type info here. Let receiver make required format changes, not sender.

File: src-java/stats-topology/stats-messaging/src/main/java/org/openkilda/messaging/info/stats/UpdateFlowPathInfo.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+import java.util.List;
+
+/**
+ * Represents update path info.
+ */
+@Value
+@EqualsAndHashCode(callSuper = false)
+@JsonSerialize
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class UpdateFlowPathInfo extends InfoData {
```
Comment (surabujin) [inline src-java/stats-topology/stats-messaging/src/main/java/org/openkilda/messaging/info/stats/UpdateFlowPathInfo.java:38]: It is identical to the RemoveFlowPathInfo, why not pull all data fields into some common parent class?

**Review by surabujin — APPROVED** [review]

File: src-java/stats-topology/stats-messaging/src/main/java/org/openkilda/messaging/info/stats/UpdateFlowPathInfo.java (hunk 2 above)
Comment (sergii-iakovenko) [reply src-java/stats-topology/stats-messaging/src/main/java/org/openkilda/messaging/info/stats/UpdateFlowPathInfo.java]: Good point. Refactored.

File: src-java/stats-topology/stats-messaging/src/main/java/org/openkilda/messaging/info/stats/RemoveFlowPathInfo.java (hunk 1 above)
Comment (sergii-iakovenko) [reply src-java/stats-topology/stats-messaging/src/main/java/org/openkilda/messaging/info/stats/RemoveFlowPathInfo.java]: Changed.

**Review by dpoltavets — APPROVED** [review]

**Pull request merged**